### PR TITLE
Updated Mining Vendor Items

### PR DESCRIPTION
**Pull request by PerishedFraud**
Made by real miner mains, after consulting with real miner mains, for real miner mains.

(And also updated for new tgui, or so I hope.)

An updated prize selection for mining vendors - the next step in making mining less of a clusterfuck. (The first step was fixing those damn airlocks, the next will be unfucking tendril RNG).

Changes some prices to be more accurate to the item's value. Adds some cool new options across the board, which will be individually noted and explained below.. Makes item order more reasonable (healing>utility tools>mining>upgrades>suit and suit upgrades> mining utility> misc.).

[Here's an image of the updated list.](https://i.imgur.com/4lArVFx.png)

[Here's a pastebin you should read if the suit upgrade section triggers you.](https://pastebin.com/AY60dCHa)

Now then, explanation of each item's inclusion ( I had to highlight to see the list, just a heads up):

- Marker beacons: Price reduced and redundant options removed. A cool item that sees little use, perhaps this'll get a bit more out of it.
- Burn and Advanced healing kits: Despite lavaland having plenty of burn damage all around, only a brute kit was available. This solves that issue.
- Welding Mask: Offers the great utility of letting miners eat without cutting off air. Despite this, wasn't previously available to them from their own department. 
- Combat Gloves: Mostly included to give incentive for miners to not rob these from ghost spawns. While it might seem like a powerful item on paper, the easiness of obtaining them as a miner as well as the existence of insul and swat crates makes this a safe addition, especially with the price in mind.
- Resonators and pickaxes: Prices nerfed heavily. These things see no use unless you are a golem or a clueless miner, and for good reason. Hopefully this incentivises more use. The most notable price change all around (all others are very minor).
- Changed mining tools to include the proto part. At least one miner thought they were upgraded versions. They aren't.
- Streamlined upgrade naming in the vendor. It's also made clear that the chassis upgrades are just skins.
- Added xenobio suit upgrade potions. See the pastebin linked in the post if you think this is wrong. These can't be mass-produced from mining and adding them is a vital step in making mining more skill/effort/time related than the horrible RNG-fest we have right now.
- Added expanded emergency oxygen tanks. These see use with bone armor, mostly. You can find regular tanks on legion corpses. This offers an option for those with no luck finding any.
- Mining conscription kit now contains a mining ID, in case you're buying it for someone without an ID to begin with, like some ghost role spawns.
- Flight potion added as a luxury 15000 option. By the time you can get this, botany and/or xenobio can mass produce them, but in case they did not, and you don't want the robo wings, well, here you go.
- Bible added for the sole reason of purifying any soulstone you can find, without having to go all the way to chapel/library.
- Space cash now says that it gives 1000, as it should.
- Pizza as a food option.

Here's to making mining better.

 ## Changelog
:cl:
add: Added some new mining vendor items.
tweak: Changed some prices.
/:cl:

### CODE REVIEW

**Comment by katebishop49** [comment]
looks good i like

**Comment by PerishedFraud** [comment]
Travis why are you failing AGAIN it's just two files

**Comment by PerishedFraud** [comment]
Now that travis isn't crying, we wait.

**Comment by utilbird** [comment]
> Flight potion added as a luxury 15000 option. 

God please no. You only have to mine for 10 minutes with a plasma cutter for that, we don’t need more miners cheesing lava with no effort.

> Adds the infinite oxygen tank as a meme option for 20000

Should probably atomize

> Added xenobio suit upgrade potions

Why? Just ask xenobio, it was one of the few things miners would “have” do on station if they wanted more powergame gear.


Most of this stuff looks like it should be atomised, some changes look nice.. others not so much :p

**Comment by PerishedFraud** [comment]
>God please no

Do you play miner at all? "Cheesing lava" is just buying an RCD (available at the cargo lathe), or just getting robo wings. If you want to mine straight for 10 minutes (even later in the round since you had to get plasma first) just to get organic wings, that is honestly good balance? It might even be too much time for something this minor? (because, again RCD).  This remark reeks of new miner or someone who doesn't even play it.

>Why? Just ask xenobio...

Read the pastebin. It explains in excrutiating, painful detail why this is horrible design.

**Comment by utilbird** [comment]
> Do you play miner at all? "Cheesing lava" is just lathing an RCD (available at the cargo lathe),

right, I forgot that "deep lava" immunity to RCD was a downstream thing.  My bad, but RCD at least takes _some_ time and materials, whereas wings let you ignore lava and chasms completely

> If you want to mine straight for 10 minutes

...what? 10 minutes of mining isn't abnormal at all, it's what most miners (that don't exclusively powergame) do.

>  It explains in excrutiating, painful detail why this is horrible design

it's really not though, this is just miners not wanting to leave the cargo/lavaland the whole game until they get enough gear to validhunt. You don't _need_ xeno gear to do your job. You don't need a large amount to have an effect, only enough for what you want.

and yes, I do play miner. I just don't need a gamer gear dispenser that gives me stuff for little to no effort outside of what I would be doing normally. 

the PR is labled "updated mining vendor items" which is fair for the some of the description and price changes, the new shortcut items are totally uncalled for. If wings are as useless as you say then why do we need them? The only thing I can see most of these changes used for is the infamous powergame validhunter miner, which are already plentiful. 

Survival pens already make medical completely useless, We don't need that for science (outside of research) too : /

**Comment by LastCrusader105** [comment]
I’m Lastcrusader105 and THIS is my favourite PR on Beestation

**Comment by MCterra10** [comment]
lol wtf is this PR LMAO
buffing the most powerful job on the station? I mean, that's kinda based of you, but the PR is pretty cringe.

**Comment by PerishedFraud** [comment]
>right, I forgot that "deep lava" immunity to RCD was a downstream thing. My bad, but RCD at least takes some time and materials, whereas wings let you ignore lava and chasms completely

I mean, you can bridge both chasms and lava with the rcd, as as a miner, you have access to all the materials you'll ever need. Unless you somehow find yourself fighting megafauna over lava (in which case only you are to blame), there's not really a difference.

>...what? 10 minutes of mining isn't abnormal at all, it's what most miners (that don't exclusively powergame) do.

Alright. But even with 10 minutes the miner will only be able to buy one, or like two of the more expensive items, tops. It adds actual decision making, honestly.

>at's really not though, this is just miners not wanting to leave the cargo/lavaland the whole game until they get enough gear to validhunt. You don't need xeno gear to do your job. You don't need a large amount to have an effect, only enough for what you want.

and yes, I do play miner. I just don't need a gamer gear dispenser that gives me stuff for little to no effort outside of what I would be doing normally.

the PR is labled "updated mining vendor items" which is fair for the some of the description and price changes, the new shortcut items are totally uncalled for. If wings are as useless as you say then why do we need them? The only thing I can see most of these changes used for is the infamous powergame validhunter miner, which are already plentiful.

At this point I refuse to believe that you've actually read the pastebin. Probably speedread it?
Let's see here...

First of all, you're confusing convenience gear with validhunt gear. Wings and suit upgrades ain't it bub. While validhunting gear is plentiful in lavaland, meat hooks, spellblades and all, this PR adds exactly zero of it. On a personal note I don't like validhunter miners either, but that's a a matter of taste.

Next, you called the items shortcuts, which...I mean...what? Do miners naturally become scientist and do xenobiology by the end of the shift? What is it a shortcut for, my man. The ability to not depend on another department is not a shortcut, it's...the ability to not depend on another department.

You also seem to have taken the whole "need" thing quite literally. Miners need such upgrades to work fast, not to work to begin with. In fact all jobs are possible at roundstart, yeah? However as pointed out several times already, walling mining's efficiency behind either RNG or another department is shit design, especially since the two have no proper connections in game. The RNG will be dealt with later. This PR deals with the latter.

>Survival pens already make medical completely useless, We don't need that for science (outside of research) too : /

You overdose on two pens in quick succession. It's useful.

>lol wtf is this PR LMAO
buffing the most powerful job on the station? I mean, that's kinda based of you, but the PR is pretty cringe.

Opinion fucking disregarded. Miner is mid tier at best and their RNG dependancy makes them objectively worse than half of the jobs on station. Anyone who says miner is the most powerful job in the station is pretty much outing themselves as too inexperienced.

...

Listen, fellas. I'll accept proper criticism. I'll do it gladly. But if you throw bullshit at me I won't hesitate to throw it back. So far this has been a shameful display, other than the oxy thing, which has been removed from the PR.

**Comment by Acensti** [comment]
This is extremely needed in bee's miner that hasnt been touched for ages

**Comment by Archanial** [comment]
Let's powergame miner more
> xeno potions in vendors

no

**Comment by PerishedFraud** [comment]
> Let's powergame miner more
> 
> > xeno potions in vendors
> 
> no

Miner's powergame does not come from xenobio's potions. It comes from RNG crap that has nothing to do with this PR.

**Comment by VictorVow** [comment]
Xenobio potions, research-locked welding masks, insuls and the bible are just further removing any need to interact with others in order to play single-player SS13 even more. You can't always reliably get these and that's the point - you need to depend on others or be reliant on a lucky spawn in the case of combat gloves/flight potion. Xeno potions in particular here are the worst additions since they make ash drake far easier. 

I could make the same arguments you made for xeno potions instead for gene powers, nanites and implants for Security but it would be similarly nonsensical. Double expanded emergency oxygen tank isn't even available to atmos/engi, this is a Deathsquad-exclusive to my knowledge and should be removed as well. 

As an ex-miner and current security main I can admit as well miners remain one of if not the most singularly powerful non-security job on the station. Hiero staff and drake armour are very powerful compared to their ease with practice and some tendril loot is similarly OP. 

Mining is already badly designed in that it just encourages solo gameplay for an hour at a time whilst you're distant in a location where no other crew can interact with you, but this further increases that. Buffing miners is not the needed change for the job, the rest of the changes outside those 5 things however is fine.

**Comment by PerishedFraud** [comment]
You seem to be the first person who's put up a truly thoughtful reply, so I thank you for that.

I am noticing one thing throughout your post, which is judging items based on availability instead of their actual usage/strength.

Indeed, based on availability alone, this would be a very silly PR. 

However, based on what the items actually do, far less so . I would even call it entirely reasonable. (it was what I was looking at, after all).

You also seem to greatly overrestimate the power of a miner without RNG loot. A miner with hierophant and drake armor is a low level threat at best and a complete joke to competent crew. Only if they get the specific pvp items like the spellblade, hook, katana and so on, do miners become a proper threat...but these are all locked behind randomness. Can you call miners powerful when the same miner can end up better than sec or worse than a debtor by doing the same things in two shifts? I wouldn't say so. (Sidenote: did I mention the RNG in this job is awful?)

You also can't actually make that nanite/genetics argument. All crew members recieve equal benefit from superpowers and nanites. Sec is a neat example on paper because it fights more often, but we all know dying or being in a deadly situation can happen to anyone. What I'm saying is that these are wide examples and generic use. Indeed, even miners benefit from these. I made a direct comparison in my pastebin to clarify this. Anyone can, for example, use a chem dispenser. It benefits everyone. However, give it to a botanist and they'll likely use it to do things other crew members won't. Similarly, if you fireproof a suit, the entire crew gets fire protection, but miners also get storm protection. Lavaproofing is entirely useless for non-miners as well, save for rare gateway examples. (The inclusion of speed and spaceproofing in the PR is admittedly there for another reason - to not force miners to always use the same suits, instead allowing them to upgrade whatever they can find/get). TLDR is that miners benefit more from xenobio's potions than any other job. If RNG doesn't favor them, their efficiency outright depends on xenobio.

But that is where my arguments end, because the rest is up to design philosophy. I can safely say that my thoughts on design philosophy are backed with a sufficient amount of miner hours and experience with the crew, but it's still a philosophy and not a fact, so bleh.

I think that anything which forces interaction, and forces player action at all (aside from just doing your job) is bad design. To clarify, the miners won't go to xenobio to have a nice chat or because they had a nice idea, they'll go to get their upgrades. This is forced. I'd go as far to call it a fake interaction, because they don't care about the scientists, rather only their produce.

But, you can also argue that forcing interactions increases gameplay opportunities and makes mining less isolated. It's not an opinion I'd agree with. I think the bad designs outweigh the good. But it's a perfectly valid opinion.

It's also not up to me to decide if items are judged by power or rarity. The double oxy tank is a perfect example. Power-wise it's just a very nifty oxy tank. Rarity-wise it's outrageous. 

Once again I'd disagree with anyone who thinks rarity should dictate changes. After all, these are changes to the game, why should they care about maintaining a random status quo? But this is also just an opinion.

One thing is utter bullshit though. the "research-locked welder mask" line. You wot mate? The KA upgrades are locked too but there they are in the vendor.

**Comment by PerishedFraud** [comment]
Double post but I'm hoping to get people to look at things more objectively instead of just comparing or trying to uphold various transient standards that aren't actually a part of the game.

Say this PR gets added.
What harm do you see coming from it?
Can you actually picture miners making the round less fun for others because of these items?
When you see a miner being a shitter, are they using tendril/fauna loot or the items they'd get from this?

I can't picture it. I can't see the welding mask, gloves or the opportunity to get a nice suit without RNG turning a miner into a shitter. The connection just isn't there.

On an unrelated note: What is the actual fucking intent of mining/xenobio? If you want to make the departments related, why are there random suits that make xenobio useless? if you want to make them isolated, why can't miners have upgrades without them?
Fuck RNG. RNG is the worst possible solution to these things. Take a side. Either make xenobio the only way miners get upgrades or let them have their alternative.

**Comment by PerishedFraud** [comment]
Look at those thumbs down. Proud of you guys. Let's keep the miners from having reasonable progression while the chemist can one-shot everyone and the scientist can blow up a third of the station.

We must keep the miner supressed in their RNG hell.

**Comment by ghost** [comment]
Ah yes, miners aren't strong enough until they literally become the doomslayer.

**Comment by PerishedFraud** [comment]
Giving miners an alternative to items they can already find if they get good RNG won't make them any closer to the doom slayer than they already are.

**Comment by ghost** [comment]
w

> Giving miners an alternative to items they can already find if they get good RNG won't make them any closer to the doom slayer than they already are.

The reason that this RNG system exists is because miners would be even more insanely busted then they are right now if it didn't, mining is a joke in terms of difficulty the scariest thing there is a chasm.

**Comment by PerishedFraud** [comment]
RNG is not the answer though. I'd much rather see something like this combined with a more generic miner nerf than making miners randomly go from OP and shit from round to round.

The next PR I have planned will outright remove the tendril RNG anyway, with tendrils dropping items you exchange for the loot, with actual prices based on power. Maybe people would accept something like this once miners aren't randomly going from doomslayer to pipsqueak.

**Comment by ghost** [comment]
Needless to say i highly doubt this'll get merged.

**Comment by PerishedFraud** [comment]
If people have a sense of actual balance it might. I can literally guarantee that no harm will come from it if it gets merged.

Sadly that might be asking too much from people who think miner is one of the stronger jobs in the game.

**Comment by St0rmC4st3r** [comment]
I have read the pastebin and belive that you are approaching the whole issue from the opposite way that should be picked. 
The entire lavaland's development history is based around misunderstanding and poorly though-out buffs and nerfs.

Let's start with the resonator. It was added at the times of slow mining, where it even had to be nerfed, despite being nowhere as nearly powerful as current plasma cutters, that are de-facto the standard mining tool nowadays. The resonator can be buffed to half-decency (which I plan to do), but it still doesn't cover the issue we are facing.

The plasma cutter was initially a pickaxe that doubled as a welding tool. (I won't dig for the PR that changed it for now.) After the rework into a projectile gun, it accepted plasma sheets as fuel like it does now. After all of this, it went downhill, since the plasma cutter makes all of the other mining equipment obsolete. It is the ultimate zero-effort traversing tool. You just shoot, run and the plasma (almost) never ends. The stormproof/lavaproof potions exist to promote interdepartmental cooperation, or.... It was so before the introduction of the ore silo that made miners just mine instead of exchanging "I will give you mats if you give me the potions." In the changes you are proposing, you are removing the bastardized leftover interaction, but you are making the problem more severe by disconnecting mining efforts from the rest of the station. It is already a game in a game as it is. We need to get further away from this state.

The convenience part makes the problem worse since it puts all the "inter-department cooperation rewards" into the lavaland bubble, where everything you would ever need, can be obtained inside of it.

>Look at those thumbs down. Proud of you guys. Let's keep the miners from having reasonable progression while the chemist can one-shot everyone and the scientist can blow up a third of the station.

This is called powercreep. You are buffing a department because other departments are very powerful, which will lead to buffs in other departments and so on. 


Some info on feature dates.
Original /tg/ commit that added the resonator: 2133f12dc4be4c7b5c8b874e1f8dc1adf543fd2b Jan 31, 2014
Original Lavaland PR: Lavaland 16033 Mar 14, 2016
Original ore silo PR: [Ready] Communal Resource Storage II 39118, Jul 25, 2018

**Comment by PowerfulBacon** [comment]
> Marker beacons: Price reduced and redundant options removed. A cool item that sees little use, perhaps this'll get a bit more out of it.

Makes sense

> Burn and Advanced healing kits: Despite lavaland having plenty of burn damage all around, only a brute kit was available. This solves that issue.

They start essentially with 2 brute and burn heals, but I guess it makes sense.

> Welding Mask: Offers the great utility of letting miners eat without cutting off air. Despite this, wasn't previously available to them from their own department.

What? I don't get this one at all. What does a welding mask have to do with miners needing to cut off their air to eat?

> Combat Gloves: Mostly included to give incentive for miners to not rob these from ghost spawns. While it might seem like a powerful item on paper, the easiness of obtaining them as a miner as well as the existence of insul and swat crates makes this a safe addition, especially with the price in mind.

This is a syndicate item and shouldn't be sold by Nanotrasen vendors and is just straight up powergame.

> Resonators and pickaxes: Prices nerfed heavily. These things see no use unless you are a golem or a clueless miner, and for good reason. Hopefully this incentivises more use. The most notable price change all around (all others are very minor).

These still won't be used by miners for good reason, they can get better tools for free. By making the costs cheaper, you are buffing golems without really affecting miners at all.

> Changed mining tools to include the proto part. At least one miner thought they were upgraded versions. They aren't.

I don't really understand what this one means

> Streamlined upgrade naming in the vendor. It's also made clear that the chassis upgrades are just skins.

Makes sense

> Added xenobio suit upgrade potions. See the pastebin linked in the post if you think this is wrong. These can't be mass-produced from mining and adding them is a vital step in making mining more skill/effort/time related than the horrible RNG-fest we have right now.

What. This makes absolutely no sense at all, why would items available only to xenobiology that requires cooperation between departments suddenly not be needed.

> Added expanded emergency oxygen tanks. These see use with bone armor, mostly. You can find regular tanks on legion corpses. This offers an option for those with no luck finding any.

I guess

> Mining conscription kit now contains a mining ID, in case you're buying it for someone without an ID to begin with, like some ghost role spawns.

Could just get an ID from HOP, but sure.

>  Flight potion added as a luxury 15000 option. By the time you can get this, botany and/or xenobio can mass produce them, but in case they did not, and you don't want the robo wings, well, here you go.

Again, a rare / unique item that shouldn't be straight up buyable.

> Bible added for the sole reason of purifying any soulstone you can find, without having to go all the way to chapel/library.

What, that is literally the ONLY job the chaplain has and you just removed it.

>Space cash now says that it gives 1000, as it should.

Fine

> Pizza as a food option.

 This is fine


Overall this PR seems like a way to just not interact with other departments, increase the powergame level of miners and make them independant from the rest of the station. Bad changes in my opinion.

TLDR; Don't add unique items from other departments to the mining vendor, because you can't be asked to wait for 50 minutes for an overpowered xenobiology item and would rather just buy it yourself.

**Comment by St0rmC4st3r** [comment]
Also, if you want to bubble it, maybe you should separate lavaland and the station a bit more? Like, make the shuttle go only at certain points of time, add a new job that will deal with getting miners stuff from the station or something?

And on the note of stormproof potions: storms were added as a serious hazard to threaten miners at all times and you are adding a quick and guaranteed option to forget about them.

**Comment by ghost** [comment]
> Also, if you want to bubble it, maybe you should separate lavaland and the station a bit more? Like, make the shuttle go only at certain points of time, add a new job that will deal with getting miners stuff from the station or something?

This is the pilot officer on colonial marines except of miners and ores you're transporting marines and intel, but it would be even more mindnumbingly boring here than it's on CM.

**Comment by PerishedFraud** [comment]
>I have read the entire pastebin...etc

That's...actually interesting. It's making me think that we should host a long-lasting vote to see what our miners actually want: isolation or more interaction?

It can be done either way.  If we get creative we could solve it from the other direction by giving actual incentive to do station-side stuff as a miner that isn't just collecting items for yourself. However one has to know whether people actually want this.

The comparison to scientist and chemist was made on a purely offensive level, btw. This PR adds utility to miners. Powerful utility, but utility nonetheless. Not things you'd actively use for PVP. Despite this the number one response was that miners were already super powerful and the most powerful job in the game. It's just pointing fingers at that and laughing. Fuck power creep.

>Welding mask

You can eat through the welding mask while internals are on! Did you not know?

>Combat gloves are a syndicate item

You can order several of them in a swat crate for 8k iirc (along with masks, spaceproof armored suits and belts). Comparing prices leaves the ones miners could buy with this greatly overpriced (but hey, they don't wait 1-2 minutes for them or need crate access).

>Don't understand what this one means

The items were just called "Kinetic Crusher" and "Kinetic accelerator" in the vendor. This lead to some miners thinking they were the "complete" versions of the proto ones they'd start with. They're identical.

>Cooperation with xenobio.

This is not a thing. "Cooperation" as it stands now is just miners asking for free shit and getting it if xenobio exists. Sometimes breaking in to get it if they're being asses. I'm all for actual cooperation but I'd argue isolation is better than what we have rn.

>Only job chaplain has

You can print infinite bibles from the library. Only people who didn't know about this would look for the chaplain for the purification. Nerf library? Make purification chaplain only? Actually that last one makes sense. Why can random people purify it?

>A rare/unique item shouldn't be buyable

Not sure I'd call it too rare since two departments can mass-produce it, whereas this is just a bottle. Still, it's one of the most haphazard parts of the PR and I wouldn't hesitate in removing it if it's a big issue for some reason.

**Comment by St0rmC4st3r** [comment]
>Btw, what's your planned resonator buff? I had an idea earlier where you could increase the aoe of it if you use it on the same tile. What did you have in mind?

Being able to detonate sonic blasts on use. Makes it more viable as a weapon, since you can hit the turf in front of the mob and step back. I already made the code, but hesitated to PR it, since I wanted to make it more of an overhaul with nerfing the cutter to eat more plasma (come on, do you even recharge it in the current state?).

**Comment by PerishedFraud** [comment]
Good stuff on both ends. Cutter is definitely bonkers and resonator needs all the love it can get.

Right now I'm thinking of making a new PR that's basically this without the xenobio stuff. 

Whether or not the potions would get included later would then depend on plenty of things. Like trying out that miner survey on isolation/cooperation. There's also that blobby xenobio rework that's upcoming.

One thing I'd like to do before that is get a good replacement for the flight potion though. What's a good luxury item for miners? Something with an actual purpose, not like the capsule...

**Comment by VictorVow** [comment]
> I am noticing one thing throughout your post, which is judging items based on availability instead of their actual usage/strength.

They're mostly small beneficial boosts which save time and add up to make miners more of a swiss-army knife threat. For example malf silicons are less likely to be able to door shock because miners would now have easier, reliable access to combat gloves. Whilst it's only a small additional boost, miners are generally one of the hardest crew to assassinate already which means these additions are unwarranted. But also xenobio potions in particular are a significant boost, since they make megafauna easier. 

> A miner with hierophant and drake armor is a low level threat at best and a complete joke to competent crew.

A low level threat? Compared to what, a wizard? This tier gear can robust traitors, changelings, xenomob armies, space dragons, etc. Drake Armour has basically the same stats as ERT hardsuits. Megafauna gear can in no way be construed "low level" unless you think miners deserve to be compared against the strongest antags, rather than crew content. 

> You also can't actually make that nanite/genetics argument. All crew members recieve equal benefit from superpowers and nanites. Sec is a neat example on paper because it fights more often, but we all know dying or being in a deadly situation can happen to anyone.

Whether all crew can get *some* benefit from it is irrelevant. Security gains the most benefit because they're the ones who are expected have to combat `carbon` mobs regularly as part of their job. X-ray vision, anti-drop implants, anti-stun nanites, etc are all *useful* to crew but not **important** in the same way it is for Security. 

Similarly fireproofing and lavaproofing is useful for crew but not **important** for them in the same way as they are for Miners, who are expected to fight `simple_animals` in a uniquely hazardous environment. But I'd still not argue that either job should gain such easy, reliable access to powerful important buffs.

> I think that anything which forces interaction, and forces player action at all (aside from just doing your job) is bad design. 

This is where we're just going to have to disagree. Interactions are important because this isn't a single-player game, it's one where there should be plentiful opportunities to both share a conversation as well as be murdered by an enterprising antagonist. Living in a bubble in Lavaland to later come back to the station at round-end with the most powerful gear doesn't offer any of these opportunities.

> the "research-locked welder mask" line. 

Just make an upgraded mining mask that allows you to eat through it. It doesn't have to be this welding mask when miners have no need for welding anyway. I think mining mask is armoured as well. 

> The next PR I have planned will outright remove the tendril RNG anyway, with tendrils dropping items you exchange for the loot, with actual prices based on power. 

RNG is there because nobody has made a better solution to miners having access to such powerful gear, but feel free to take a stab at it by reworking tendril loot.

**Comment by PerishedFraud** [comment]
>A low level threat? Compared to what, a wizard? This tier gear can robust traitors, changelings, xenomob armies, space dragons, etc. Drake Armour has basically the same stats as ERT hardsuits. Megafauna gear can in no way be construed "low level" unless you think miners deserve to be compared against the strongest antags, rather than crew content.

Okay! Miner with hierophant and drake armor! Let's compare to some crew members!

Miner with hierophant has a closed-space dps advantage. That's nice but requires good setup. Comparable to surprise push stunlocking.
Miner with hierophant's ranged attacks can be reliably dodged every time!

VS Security: Security fires disablers at miner with hierophant. Miner with hieropahnt is forced to retreat or loses. He simply cannot kill sec before they shoot him to horizontal even if they somehow don't dodge.
VS command: Telescopic baton and shove combined with another weapon, preferrably a gun. Miner with hierophant is down as soon as he entered melee range. He has to put up with gunfire otherwise, a losing fight!
VS anyone with a fireaxe, spear, sabre, etc: As long as the fight is not in a confined space, miner with hierophant LOSES unless the opponent is slowed by something.

In other words: more powerful than average crewman but not even close to anyone geared to fight players. Mobbing gear just doesn't compare. Let's not even get the real powerhouses like chem or xenobio here.

>Upgraded mining mask
Actually a cool idea. Sure. I'll do it. Adds more flavor to miners too I guess.

>No better solution to RNG
I am honestly 100% confident in the solution I'll have for tendrils. It's gonna take plenty of work to get there but it won't be a buff or a nerf, and I think most people will find it cool. It's far less risque than this PR in concept.

>Important vs useful

Not much to add here beyond what's already said. We have to actually know whether miners want to be isolated or integrated with other players before further decisions about this can be made. I've assumed they'd prefer the former but here we are. 

An interesting alternative to outright giving miners xenobiology's upgrades would be simply having an upgraded suit available in the vendor though. A suit that doesn't have as much armor as the tendril suits, for example, but does offer the other benefits of being spaceproof and fireproof. Would this suit be worth 10k or perhaps 20k points? Food for thought.

Your argument for the minor boosts is entirely legitimate, by the way.  I have no counterpoint to it. It's just true.

**Comment by utilbird** [comment]
*cry. short novel

TL;DR, shortcut gear bad. Effort for combat strength is unbalanced. This PR is unneeded and just makes mining even _more_ isolated.

> VS Security: Security fires disablers at miner with hierophant. Miner with hierophant is forced to retreat or loses. He simply cannot kill sec before they shoot him to horizontal even if they somehow don't dodge. In close combat, he loses to stun baton but can win by getting an extra hit in or so.


> VS command: Telescopic baton and shove combined with another weapon, preferrably a gun. Miner with hierophant is down as soon as he entered melee range. He has to put up with gunfire otherwise, a losing fight!


> VS anyone with a fireaxe, spear, sabre, etc: As long as the fight is not in a confined space, miner with hierophant LOSES unless the opponent is slowed by something.

all of these examples are assuming the opponent get's the alpha strike and is ignoring that the miner has gamer armour. Also, the hiero staff can really fast, if you are attacking someone who has a spear, you are doing the ~15 damage from hitting them with the club PLUS the burn damage from the blast radius. If the opposing attacker is actually robust and dodges every single hit then sure, although miners almost always have more than just drake armour + a single weapon. I think the staff is pretty shit compared to some other megafauna gear anyway. Go get the god tongue and win every fight with the sleep command, or just get a meat hook from a chest.

Let's change the situation to say, a wizard round. 

Miner just came back from station with his free ERT armour he got from shooting a spiky ground dildo for 10 seconds, he is now completely immune to what is supposed to be one of the strongest antags in the game. combine that with nothing more than a laser cutter and he probably just ended the round

Too specific?

Miner exits lavaland with his shiny new antique laser gun, goliath upgraded armour and let's sayy.. The easy access slow-aheal pens he bought 10 of from the vendor. Miner hears the satisfying announcement of a traitor's location, in the not-so unlikely event that the miner finds said traitor, all he has to do is start clicking the valid with his infinite egun or click himself with a medi-pen in the rare event that the surprised traitor managed to get him below 70% health before going horizontal.

"but this applies to all jobs"

technically yes, but the amount of effort required to do the same with _most_  jobs isn't really comparable. Mining rewards you with gamer gear just for doing your job (this PR would make it even worse) whereas almost every other job you have to do something else that is semi-related to your main job to get similar (but usually much, much weaker) gear.

Mining is inherently flawed, a complete rework from the ground up would be needed to resolve the issues it has. Adding snowflake/shortcut items doesn't do anything to solve the problem besides "xenobio didn't do x instantly" or bad RNG with powergame tendrils. What's the point of rewarding miner's with something that for the most part, actively encourages them to _not_ do the job, but now I'm getting off topic.

> I am honestly 100% confident in the solution I'll have for tendrils. It's gonna take plenty of work to get there but it won't be a buff or a nerf, and I think most people will find it cool. It's far less risque than this PR in concept.

go do that then instead of this : /



it's 3am and I need slep ;-;

**Comment by PowerfulBacon** [comment]
add the supermatter sword for 450 mining points

**Comment by triplezeta** [comment]
So I was made aware that this PR was being made some time yesterday, and when I saw the new equipment list I almost passed out from laughter. 

Let's start with the xenobio potions.
4000 points for removing slowdown? Really? And 4000 for fireproofing, 6000 for pressureproofing... I understand your frustration with needing to depend on xenobio in order to get these things, but I have a much simpler solution for you: don't. None of these things are required for a miner in order to be prepared for megafauna fighting. ALL suits available to miners with the only exception being the mining hardsuit do not have slowdown. ALL suits available to miners have innate fireproofing (but not temperature-proofing. That's what the leporazine in survival medipens is for.) Spaceproofing and lavaproofing your suit is *nice*, but absolutely not required for a miner to do regular miner things.

With flight potion, again, it's very nice for miners to have, but absolutely not required. It's a luxury, like everything else you've added, and one that's supposed to be very rare.

The welding gas mask and combat gloves... miners don't need either of these! Miners don't need shock insulation, and if they want it, they can get it from the *department they're supposed to be working for*. If they want a welding gas mask, maybe they should try asking science or engineering nicely?

Nothing you've added is something that a miner desperately needs, or should be able to get easily. 

EDIT: 
> Miner with hierophant has a closed-space dps advantage. That's nice but requires good setup. Comparable to surprise push stunlocking.
Miner with hierophant's ranged attacks can be reliably dodged every time!

> VS Security: Security fires disablers at miner with hierophant. Miner with hierophant is forced to retreat or loses. He simply cannot kill sec before they shoot him to horizontal even if they somehow don't dodge. In close combat, he loses to stun baton but can win by getting an extra hit in or so.
> VS command: Telescopic baton and shove combined with another weapon, preferrably a gun. Miner with hierophant is down as soon as he entered melee range. He has to put up with gunfire otherwise, a losing fight!
> VS anyone with a fireaxe, spear, sabre, etc: As long as the fight is not in a confined space, miner with hierophant LOSES unless the opponent is slowed by something.

mfw the guy buffing mining is too unrobust to be able to properly murderbone with one of two ranged weapons that can ignore sightlines completely

**Comment by utilbird** [comment]
If you need to spend this long arguing why your PR is a good idea. It probably not a good idea

**Comment by PerishedFraud** [comment]
In this PR: People reveal how much they hate mining, and probably something about the state of the game in general. I'm not gonna get into that though.

**Comment by MCterra10** [comment]
Here's the simple rule of thumb that has never failed me thus far:
If you have to justify your PR with the equivalent of a small essay, it's probably shit.

That's how it was for cobbychem, that's how it was for mining point removal, that's how it was for singulo and tesla removal, and that's how it is for this miner buff.

**Comment by PerishedFraud** [comment]
> Here's the simple rule of thumb that has never failed me thus far:
> If you have to justify your PR with the equivalent of a small essay, it's probably shit.
> 
> That's how it was for cobbychem, that's how it was for mining point removal, that's how it was for singulo and tesla removal, and that's how it is for this miner buff.

The OP is enough to justify it. If your argument against me is that I have the decency to explain things in detail instead of calling people shit, I don't know what to tell you.

**Comment by rabbetbunzel** [comment]
stupid !! failcringe

**Comment by rabbetbunzel** [comment]
why would you think this was a good idea

**Comment by triplezeta** [comment]
add flightsuits for 1000 points :moyai:

the ultimate utility, right?

**Comment by PerishedFraud** [comment]
> stupid !! failcringe
> why would you think this was a good idea

Fuck off troll.
